testing/driver/monkey/Make.defs

```diff
@@ -1,5 +1,5 @@
 ############################################################################
-# apps/testing/monkey/Make.defs
+# apps/testing/driver/monkey/Make.defs
```

Comment:
monkey is more like a command line, let's move to testing/tools/ or testing/utils

Reply:
This also looks like input device drivers test so drivers location looks good? We talked about this case in https://github.com/apache/nuttx-apps/pull/2931 :-)

Reply:
please reference this wiki: https://en.wikipedia.org/wiki/Monkey_testing
to understand why we call it monkey, not input.

Reply:
As above, looks like need to split testing utilities and tests scenarios?